Mage.Tests/src/test/java/org/mage/test/cards/enchantments/AnimateDeadTest.java

```diff
@@ -192,6 +193,7 @@ public void testAnimateAndDragonlordAtarkaWithNoTargets() {
         addCard(Zone.GRAVEYARD, playerB, "Dragonlord Atarka", 1);
 
         castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, "Animate Dead", "Dragonlord Atarka");
+        addTarget(playerA, TestPlayer.TARGET_SKIP);
```

Comment:
Looks like something changed in the targets logic, so Animate Dead test failed after refactor (now it’s require to choose target instead ignore it), usage details: #4428

Reply:
Possible reason: changes in the params (old code used param for max number and default 1 for min number, current code use param for max and min number). Need additional checks — is it by design or it’s a bug.

![IMG_1005](https://github.com/user-attachments/assets/b59fef8b-392a-42b7-a960-8624da449e1c)

Reply:
Well, looks like it’s a bug cause 107.1c require a non-zero number of targets (e.g. min number must be 1).

Reply:
I believe everything should be correct. At the time it was implemented, Dragonlord Atarka (among other cards) did require at least one target under the old rule 107.1c. When I went through all the cards affected in this refactor, I did change Dragonlord and others to have zero minimum targets instead of one, under the new rule.

Reply:
Yes, although I think there were even more cards with the opposite issue (e.g. "one or two targets" allowing zero targets), so that should be noted as well.